packages/llamaindex/src/vector-store/LibSQLVectorStore.ts

```diff
@@ -0,0 +1,283 @@
+import {
+  BaseVectorStore,
+  type MetadataFilters,
+  type VectorStoreBaseParams,
+  type VectorStoreQuery,
+  type VectorStoreQueryResult,
+} from "./types.js";
+
+import { createClient, type Client, type InStatement } from "@libsql/client";
```

Comment:
further more, I want support also sqlite, sqlite3, node:sqlite, bun:sqlite...

Reply:
This PR is focused on the local and remote libSQL instances. I'm happy to help with the sqlite variants, but in another PR. Does that work for you?

Reply:
It's ok to start like this, but we need to put the store into its own npm package